src/Utils.js

```diff
@@ -0,0 +1,48 @@
+import $ from 'jquery';
+import { proxy, minYear, maxYear, defaultYear } from './config';
+
+function parseHtml(html) {
+  let parser = new DOMParser();
+  return parser.parseFromString(html, 'text/html');
+}
+
+exports.fetchDocument = function(url) {
+  return fetch(url)
+    .then((response) => response.json())
+    .then(html => parseHtml(html))
+}
+
+exports.parseInput = function() {
```

Comment:
I don't get this function at all 😵 💫 🤒 
According to its name it "parses" input but reading its code I've figured out it actually  reads input and triggers UI changes according to received input which should be, by definition, implemented in separate function.